src/core/server/plugins/discovery/plugin_discovery.test.ts

```diff
@@ -26,7 +26,7 @@ import { Config, ConfigService, Env, ObjectToConfigAdapter } from '../../config'
 import { getEnvOptions } from '../../config/__mocks__/env';
```

Comment:
__note:__ hmm just realized `plugin_discovery.test.ts` is missing `s` after `plugin` to be consistent with `plugins_discovery.ts`, likely was already in my initial plugin discovery PR :see_no_evil: